src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+public class Rectangle extends Shape {
+    private double height;
+    private double width;
+
+    public Rectangle(String color, double height, double width) {
+        super(Colors.valueOf(color));
```

Comment:
The `Colors.valueOf(color)` assumes the color string is always a valid enum constant. Consider adding error handling to manage cases where the color might not be valid.